frontend/src/pages/ClientOverviewPage.tsx

```diff
@@ -115,6 +124,11 @@ const ClientOverview: FC<{
         metrics={metrics}
         data={clientData}
       />
+      {representatives.length && (
```

Comment:
```suggestion
      {representatives.length > 0 && (
```

This would display literal "0"

Reply:
Definitely made this mistake before too, thanks for catching it